### DISCUSSION TITLE
yarn dev not launching app locally

### DISCUSSION THREAD

**Discussion by GoonerBrian** [responsively-org/responsively-app#660] [Q&A]
This is my first time contributing and I'm having trouble getting the app to run locally. I'm running `node v17.1.0` and `yarn 1.22.17`. When I navigate to the `desktop-app` folder and run the `yarn dev` command I get a `error Command failed with exit code 127.` message. Can anyone help me out with this?

**Comment by jjavierdguezas** [comment]
Did you install first (```yarn``` or ```npm install```) to get all dependencies?

**Comment by RibosomesandRocketships** [comment]
Necro'd - I am having trouble getting yarn dev to run as well (and I did run yarn first to get dependencies). 
```
Error: cannot find module 'common/constants'
...
[electronmon] ignoring exit with code 1
```

However, it's there when I check the files. Not sure how to proceed, any help very welcome! Seems to just freeze my terminal.
